datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -18,6 +18,7 @@
 use datafusion::config::ConfigOptions;
 use datafusion::optimizer::analyzer::expand_wildcard_rule::ExpandWildcardRule;
 use datafusion::optimizer::AnalyzerRule;
+use substrait::proto::expression::field_reference::{RootReference, RootType};
```

Comment:
nit: looks like the imports are kinda messy anyways but mind putting this somewhere next to other substrait things still?

Reply:
Sure;  I'd just let vscode sort out the imports; but yes can move it down.